src/test/java/org/fundacionjala/coding/nestor/FizzBuzzTest.java

```diff
@@ -0,0 +1,73 @@
+package org.fundacionjala.coding.nestor;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
+ * Test class Fizz Buzz.
+ */
+public class FizzBuzzTest {
+  private FizzBuzz fb;
+
+  /**
+   * Method Before setup.
+   */
+  @Before
+  public void setup() {
+    fb = new FizzBuzz();
+  }
+
+  /**
+   * Method general testing.
+   */
+  @Test
+  public void fizzBuzzTest() {
+    final Integer[] counter = new Integer[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20};
+    String[] expected = new String[]
+        {"1", "2", "Fizz", "4", "Buzz", "Fizz", "7", "8", "Fizz", "Buzz",
+            "11", "Fizz", "13", "14", "FizzBuzz", "16", "17", "Fizz", "19", "Buzz"};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "FizzBuzz" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)